spec/services/slack_spec.rb

```diff
@@ -4,7 +4,7 @@
 describe Service::Slack do
   let(:config) do
     {
-      :url => 'https://slack_team.slack.com/services/hooks/incoming-webhook?token=token',
+      :url => 'https://crashtest.slack.com/services/hooks/incoming-webhook?token=token',
```

Comment:
The spec allowed a Slack::Notifier to be initialized and it didn't like the domain with an underscore in it, so I switched it to be a happy fake domain name.